src/lib/storage/table.cpp

```diff
@@ -132,6 +133,17 @@ std::shared_ptr<const Chunk> Table::get_chunk(ChunkID chunk_id) const {
   return _chunks[chunk_id];
 }
 
+void Table::delete_chunk(ChunkID chunk_id) {
```

Comment:
I believe "remove" is more commonly used. Let's try to reserve "delete" for things that correlate with SQL's delete.